rfc/2022/2022-09-15_Design-System-Team_Migrate-vets-website-off-of-formation.md

```diff
@@ -0,0 +1,106 @@
+# RFC: formation and USWDS1 will be removed from `vets-website` and `content-build` and will instead use the Design System's new `css-library` based on USWDS3
+
+* Comment Deadline: `2022-10-11`
+* Team Crew and Name: Platform/DST
+* Authors:
+  * [Brooks Johnson](https://github.com/bkjohnson)
+  * [Contributing Author 1 Github Handle](https://github.com/ContributingAuthor1Handle)
+  * [Contributing Author 2 Github Handle](https://github.com/ContributingAuthor2Handle)
+* [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-arch/pull/-1)
+
+## Background
+
+`formation`, the CSS framework for va.gov, is dependent on USWDS version 1 as is the rest of va.gov. This is now a dead branch and the latest version of USWDS is version 3.
+In order to continue to receive updates from USWDS, we need to switch to a more recent version.
```

Comment:
"In order to continue to receive updates" and bug fixes from USWDS, and to be able to contribute back, we need...

Reply:
Fixed: https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/pull/48/commits/64eb3d4245eb71bdb2ac706cd51fcc626d4825ff